### PR TITLE
clang fix

### PR DESCRIPTION
**Pull request by HarithaIBM**

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -1,11 +1,12 @@
 export ZOPEN_TYPE="TARBALL"
 
 export ZOPEN_TARBALL_URL="https://gnu.askapache.com/diffutils/diffutils-3.8.tar.xz"
-export ZOPEN_TARBALL_DEPS="curl xz make zoslib"
+export ZOPEN_TARBALL_DEPS="curl xz make zoslib bash coreutils"
 
-export ZOPEN_EXTRA_CFLAGS="-Wno-unused-const-variable -Wno-tautological-constant-out-of-range-compare -Wno-missing-prototypes -Wno-tautological-compare -Wno-implicit-function-declaration"
+export ZOPEN_EXTRA_CFLAGS="-Wno-unused-const-variable -Wno-tautological-constant-out-of-range-compare -Wno-missing-prototypes -Wno-tautological-compare -Wno-implicit-function-declaration -O0"
```
Comment (IgorTodorovskiIBM) [inline buildenv:6]: Validate if -O0 is needed

Comment (HarithaIBM) [reply buildenv]: Right, it is not required, i have used fno-builtin

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Review by MikeFultonDev — APPROVED** [review]
approved. I noticed @IgorTodorovskiIBM restricted to *alloc builtins being turned off rather than turning all builtins off for another package - would that be worth trying too or is this a different issue?

**Pull request merged**